release-schema.json

```diff
@@ -1,29 +1,59 @@
+          "$ref": "#/definitions/SecondStage"
+        }
+      }
+    },
     "Lot": {
       "properties": {
         "secondStage": {
           "title": "Second stage",
           "description": "Information about the second stage of a two-stage procedure (e.g. a restricted procedure, a competitive procedure with negotiation, a competitive dialogue or an innovation partnership).",
-          "type": "object",
-          "properties": {
-            "minimumCandidates": {
-              "title": "Minimum number of candidates",
-              "description": "The minimum number of candidates to be invited for the second stage of the procedure. If there is an exact limit on the number of candidates, set `minimumCandidates` and `maximumCandidates` to the same number.",
-              "type": [
-                "number",
-                "null"
-              ]
-            },
-            "maximumCandidates": {
-              "title": "Maximum number of candidates",
-              "description": "The maximum number of candidates to be invited for the second stage of the procedure. If there is an exact limit on the number of candidates, set `minimumCandidates` and `maximumCandidates` to the same number.",
-              "type": [
-                "number",
-                "null"
-              ]
-            }
-          }
+          "$ref": "#/definitions/SecondStage"
+        }
+      }
+    },
+    "SecondStage": {
+      "title": "Second stage",
+      "description": "Information about the second stage of a two-stage procedure (e.g. a restricted procedure, a competitive procedure with negotiation, a competitive dialogue or an innovation partnership).",
+      "type": "object",
+      "properties": {
+        "minimumCandidates": {
+          "title": "Minimum number of candidates",
+          "description": "The minimum number of candidates to be invited for the second stage of the procedure. If there is an exact limit on the number of candidates, set `minimumCandidates` and `maximumCandidates` to the same number.",
+          "type": [
+            "number",
+            "null"
+          ]
+        },
+        "maximumCandidates": {
+          "title": "Maximum number of candidates",
+          "description": "The maximum number of candidates to be invited for the second stage of the procedure. If there is an exact limit on the number of candidates, set `minimumCandidates` and `maximumCandidates` to the same number.",
+          "type": [
+            "number",
+            "null"
+          ]
+        },
+        "successiveReduction": {
+          "title": "Successive reduction",
+          "description": "The procedure will take place in successive stages. In each stage, some participants may be eliminated.",
+          "type": [
+            "boolean",
+            "null"
+          ]
+        },
+        "possiblyNoNegotiation": {
```

Comment:
I think the term `noNegotiationNecessary` has the right semantics, as I discuss in https://github.com/eForms/eForms/issues/323#issuecomment-517349757

`possiblyNoNegotiation` makes it sound like some external factors independent of the buyer/procuring entity's prerogative might affect whether there are negotiations.

Reply:
I agree my proposal is not ideal, but I still believe the current label is misleading and will mislead OCDS data consumers. I'll fix it for the sake of consistency with eForms.

Reply:
We can think of other options, e.g. `mayNotHaveNegotiation`.

Reply:
OK to implement `mayNotHaveNegotiation`?

Reply:
I can't think of a label with a better balance between consistency (with our `hasProperty` boolean fields) and clarity.

Reply:
Sounds good, let's go with that.